pom.xml

```diff
@@ -60,7 +60,7 @@
     <junit.version>4.12</junit.version>
     <powermock.version>1.6.4</powermock.version>
     <derby.version>10.10.2.0</derby.version>
-    <beeju.version>1.2.1</beeju.version>
+    <beeju.version>3.0.0</beeju.version>
```

Comment:
3.0.0 means a move to Java 8 (didn't you see my earlier comment?). Circus Train is still Java 7 (https://github.com/HotelsDotCom/circus-train/blob/master/pom.xml#L47). Can you try 2.0.0 instead?

Reply:
As discussed, updated this to instead use Beeju 2.0.0, thanks Adrian